source/plugins/go/src/ingestion_token_utils.go

```diff
@@ -78,6 +78,11 @@ type AgentConfiguration struct {
 				Containerinsights []struct {
 					ID            string   `json:"id"`
 					Originids     []string `json:"originIds"`
+					Extensionsettings struct {
+						DataCollectionSettings struct {
+							EnableContainerLogV2 bool `json:"enableContainerLogV2"`
```

Comment:
This struct other fields as well, right? Lets keep this struct consistent with other data collection settings as well.

Reply:
added